pmd-core/src/main/java/net/sourceforge/pmd/renderers/TextRenderer.java

```diff
@@ -16,6 +16,10 @@
  */
 public class TextRenderer extends AbstractIncrementingRenderer {
 
+    private static final Character SMALL_SEPERATOR = ':';
```

Comment:
```suggestion
    private static final Character SMALL_SEPARATOR = ':';
```

Reply:
Fyi, this is also the case for the other constants, I think, it's easier to correct in an IDE than in GH

Reply:
Ok, will change it